syntax = "proto3";

package CAPTCHA_RPC;

service Captcha {
  rpc GetCaptcha (Empty) returns (CaptchaJson);
  rpc VerifyCaptcha (CaptchaJson) returns (Status);
}

message Empty {}

message CaptchaJson {
    string captcha_json = 1;
}

message Status {
    bool status = 1;
}